libs/libc/uuid/lib_uuid_create.c

```diff
@@ -22,9 +22,40 @@
  * Included Files
  ****************************************************************************/
 
+#include <sys/random.h>
+#include <errno.h>
 #include <stdlib.h>
 #include <uuid.h>
 
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+static int uuid_getrandom(FAR void *buf, size_t size, int flags)
+{
+  FAR char *tmp = buf;
+  ssize_t ret;
+
+  while (size > 0)
+    {
+      ssize_t ret = getrandom(tmp, size, flags);
```

Comment:
```suggestion
      ret = getrandom(tmp, size, flags);
```

Reply:
I move the line 37 instead.

Reply:
If so, then `FAR char *pos = buffer;` should be moved :)

Reply:
we can leave as it is

Reply:
No, we can't since tmp/buf need keep the last iterator to make the next loop fill the next part of buffer.

Reply:
yes, missed that